shared/src/metabase/shared/parameters/parameters.cljc

```diff
@@ -255,8 +255,8 @@
   "Given the context of a text dashboard card, replace all template tags in the text with their corresponding values,
   formatted and escaped appropriately."
```

Comment:
Could you update the docstring to mention the new parameter? ("escaped appropriately" is no longer true in every case)

Might be worth calling out specifically that this should be `false` when the output isn't being rendered directly as markdown, such as in header cards, and `true` otherwise. Otherwise it's a bit confusing when to use it.